Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -36,6 +36,9 @@ function spawn_model() {
 
 	gz model --spawn-file=/tmp/${MODEL}_${N}.sdf --model-name=${MODEL}_${N} -x 0.0 -y $((3*${N})) -z 0.0
 
+    #For RTPS, run micrortps_agent like below
```

Comment:
This needs to me made visible in the devguide, not inside the script.

Reply:
Thanks. I will remove it.